cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,132 @@
 /// <reference types='cypress' />
 
+const faker = require('faker');
+
+const firstName = faker.name.firstName();
+const lastName = faker.name.lastName();
+const email = faker.internet.exampleEmail(firstName, lastName);
+const age = faker.random.number({ min: 21, max: 60 });
+const salary = faker.random.number({ min: 2000, max: 7000 });
+const department = faker.commerce.department();
+
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables');
+  });
+
+  it('should be the Pagination', () => {
+    cy.get('.-pagination')
+      .should('exist');
+    cy.get('.-pagination')
+      .should('contain', 'Previous');
+    cy.get('.-pagination')
+      .should('contain', 'Next');
+    cy.get('.-pageJump')
+      .should('exist');
+  });
+
+  it('should be able to chenge rows count section', () => {
+    cy.get('[aria-label="rows per page"]')
+      .should('exist');
+      cy.get('[aria-label="rows per page"]')
+      .select('10');
+    cy.get('[aria-label="rows per page"]')
+      .should('have.value', '10');  
```

Comment:
```suggestion
      .should('have.value', '10');  
```
Add an assertion here that checks table rows. Because now you select value in the dropdown and than check that this dropdown contain selected values